sdk/turing/batch/job.py

```diff
@@ -86,6 +87,36 @@ def status(self) -> 'EnsemblingJobStatus':
     def error(self) -> str:
         return self._error
 
+    def refresh(self):
+        """
+        Fetches latest updates of this ensembling job
+        """
+        from turing.session import active_session
```

Comment:
I probably missed this in the earlier reviews. I generally prefer having imports at the top level, so it is obvious what a module depends on. I've only seen this style of writing imports inside functions for optional dependencies which makes sense. Is there another reason why the active session is being imported in every function instead of globally ?

If it is because the `active_session` may not be set initially, I think it can be overcome in one of the following ways:
* `import turing.session` -> then use `turing.session.active_session` elsewhere
* Create a getter function in `turing.session` that can be imported and be responsible for returning the session object: `active_session().get_ensembling_job(...`

wdyt?

Reply:
iiric, local imports were added to avoid/solve cyclic imports in the codebase. But now I addressed it here: https://github.com/gojek/turing/pull/75/commits/d559dd0fcc36c25452c037c2d27b7650cf891705